lms/djangoapps/mobile_api/users/views.py

```diff
@@ -221,6 +221,7 @@ class UserCourseEnrollmentsList(generics.ListAPIView):
           * video_outline: The URI to get the list of all vides the user can
             access in the course.
           * id: The unique ID of the course.
+          * channel_id: A unique "clean" (alphanumeric with '_') ID of the course.
```

Comment:
Given this description, I sort of wonder if we should call it "channel_id" since that is parse specific nomenclature (the reader might ask "what the heck is a channel?"), but it doesn't actually mention parse so maybe it's fine. But if you think it's worth it, quick brainstorm for names includes: "clean_id", "safe_id", "alphanumeric_id".